topostats/utils.py

```diff
@@ -199,20 +209,68 @@ def get_thresholds(
     """
     thresholds = defaultdict()
     if threshold_method == "otsu":
-        thresholds["above"] = threshold(image, method="otsu", otsu_threshold_multiplier=otsu_threshold_multiplier)
+        thresholds["above"] = {
+            "minimum": threshold(image, method="otsu", otsu_threshold_multiplier=otsu_threshold_multiplier),
+            "maximum": np.Infinity,
+        }
+        thresholds["below"] = None
     elif threshold_method == "std_dev":
         try:
```

Comment:
This a long block of nested `if: ... else:` statements that are easy to get lost in.
Perhaps it could be broken down into a couple of methods that are easier to reason about. This would help address the linting errors that have been disabled. Perhaps `_get_mean_min_max()`.

Similarly break the code for `absolute` out to its own function too `_get_absolute_min_max()`.

Reply:
Done 👍 sorry for the delay